applications/app/controllers/InteractiveController.scala

```diff
@@ -19,6 +19,7 @@ import renderers.DotcomRenderingService
 import services.USElection2020AmpPages
 import services.InteractivePickerInputData
 import implicits.{AmpFormat, EmailFormat, HtmlFormat, JsonFormat}
+import org.joda.time.DateTime
```

Comment:
Just wanted to flag that the Joda library has been deprecated for a number of years and I know other teams (Ophan, specifically) are in the process of migrating away from it in favour of [Java's Date and Time API](https://www.oracle.com/technical-resources/articles/java/jf14-date-time.html). Even the author of Joda even suggests using the new API ([here](https://blog.joda.org/2014/11/converting-from-joda-time-to-javatime.html)). 

I wonder if importing Joda in these Scala files creates tech debt as eventually we'll need to move away from it. What are your thoughts?

Reply:
@nicl Any thoughts on this?

Reply:
Yeah, I suspect either is fine here (as we are instantiating the date from a Long/CAPI timestamp anyway). The problem is that most of the Frontend code uses Joda so, for consistency, it is usually better to stick with that. However, there are parts that use Java Time (including e.g. football stuff) too. It would be a mammoth task to migrate away from Joda in Frontend given it is everywhere.

Reply:
By the way, I started decommissioning joda from frontend: https://github.com/guardian/frontend/pull/23950 (first of many steps).